1_4.js

```diff
@@ -0,0 +1,24 @@
+let arr = [
```

Comment:
再代入が行われないので、こちらは定数で宣言することができますね。
「再代入」について一度調べてみましょう！
```suggestion
const arr = [
```

また、コードを書く際には「可読性」を意識しましょう。
自分や他の人がコードをみたときに、内容がわかりやすいかどうかです。
前回宮崎がお伝えしたように、抽象的な命名は可読性を低下させます。
ここでいうと、let arrという部分が該当します。
todoですとかlistですとかtaskなどのような具体的な名前に変更しましょう。